### PR TITLE
Hashtag matchers

### PR DESCRIPTION
**Pull request by rafaelcgo**
Adding Hashtag support.

Never done a PR before, hope it is ok.

### CODE REVIEW

**Comment by rafaelcgo** [comment]
What happened?
Something with JRuby, don't know: https://travis-ci.org/vladimir-tikhonov/regexy/jobs/54287914

File: spec/web/hashtag_spec.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# https://gist.github.com/janogarcia/3946583
```
Comment (vladimir-tikhonov) [inline spec/web/hashtag_spec.rb:3]: Please, add `# encoding: UTF-8` comment to satisfy jruby

Comment (rafaelcgo) [inline spec/web/hashtag_spec.rb:3]: What should I do? Just add the line, force commit on my repo and do another PR?

Comment (vladimir-tikhonov) [inline spec/web/hashtag_spec.rb:3]: It seems that you already do it right :+1:

**Comment by vladimir-tikhonov** [comment]
Ok, ci is green now. Thanks a lot for your work, @rafaelcgo, highly appreciated! :heart:

**Comment by rafaelcgo** [comment]
:+1: 
Glad to help.
